orgstats/stats.go

```diff
@@ -62,18 +62,23 @@ func isBlacklisted(blacklist []string, s string) bool {
 	return false
 }
 
-func (s Stats) add(cs *github.ContributorStats) {
+func (s Stats) add(cs *github.ContributorStats, weeks int32) {
 	if cs.Author == nil {
 		return
 	}
 	stat := s[*cs.Author.Login]
 	var adds int
 	var rms int
 	var commits int
+	t1 := time.Now()
+	lastWeeksDuration := time.Hour * 24 * 7 * time.Duration(weeks)
```

Comment:
I still don't get why you need to do this... the stats already come in buckets of weeks, so couldn't you just:

```go
for i, week := range cs.Weeks {
	if i > weeks {
		break
	}
	adds += *week.Additions
	rms += *week.Deletions
	commits += *week.Commits
}
```